client/types/account/account-details.d.ts

```diff
@@ -1,12 +1,31 @@
+/**
+ * Current design only uses `green`, `yellow`, `red` but add all front-end assisted options.
```

Comment:
What is the current design? IIUC those colors are coming from the backend and nobody will come here and delete this comment once they start getting used. IMO we can skip the comment(s).

Reply:
> What is the current design? 

The figma file in [WOOPMNT-5282](https://linear.app/a8c/issue/WOOPMNT-5282)